.github/actions/install-wdk/action.yaml

```diff
@@ -7,16 +7,16 @@ inputs:
     required: true
   source:
     description: "Source to install WDK from (nuget or winget)."
-    required: false
+    required: true
     default: "nuget"
   # required to set WDKBinRoot and WDKToolRoot based on the host architecture when using nuget
   host:
-    description: "Host architecture (amd64 or arm64). Required when using NuGet source."
```

Comment:
Not sure why the descriptions were changed. The new version is no clearer than what was already there. It's an unnecessary change that doesn't add value.